cluster/definition.go

```diff
@@ -21,15 +21,25 @@
 	"io"
 
 	ssz "github.com/ferranbt/fastssz"
+	"github.com/libp2p/go-libp2p-core/peer"
 
 	"github.com/obolnetwork/charon/app/errors"
+	"github.com/obolnetwork/charon/p2p"
 )
 
 const (
 	definitionVersion = "v1.0.0"
 	dkgAlgo           = "default"
 )
 
+// NodeIdx represents the index of a node/peer/share in the cluster as operator order in cluster definition.
+type NodeIdx struct {
+	// PeerIdx is the index of a peer in the peer list (it 0-indexed).
+	PeerIdx int
+	// ShareIdx is the tbls share identifier (it is 1-indexed).
+	ShareIdx int
```

Comment:
Yeah this is an existing concept, it is just the PeerIdx but 1-indexed since tbls shares are 1 indexed, this just makes it explicit when to use which index.